src/AssemblerHttpServer.cpp

```diff
@@ -325,9 +325,12 @@ void Assembler::writeNavigation(ostream& html) const
         {"Assembly graph edges support", "exploreAssemblyGraphEdgesSupport"},
         {"Compressed assembly graph", "exploreCompressedAssemblyGraph"},
         });
-    writeNavigation(html, "Help", {
-        {"Documentation", "docs/index.html"},
-        });
+    
+    if (!httpServerData.docsDirectory.empty()) {
```

Comment:
I would like to gradually switch away from `!`, `&&`, `||` in favor of keywords `not, and, or`. But it is not essential to make this change here.